docs/modules/ROOT/pages/index.adoc

```diff
@@ -2,13 +2,48 @@
 
 image::syn.png[]
 
-This is the documentation of Project Syn.
+== Introduction
 
-== Components
+> Project Syn is a pre-integrated set of tools to provision, update, backup, observe and react/alert production applications on Kubernetes and in the cloud. It supports DevOps through full self-service and automation using containers, Kubernetes and GitOps.
+
+All code lives on GitHub under https://github.com/projectsyn.
+
+=== What's in for: Dev
+
+The tooling provided by Project Syn enables the user of the platform to consume all the available services in self-service. By fully automating the day-2 operations (see below), all consumed services are automatically "managed". There is no need to use different configuration management tooling, it's all defined at the same place: Kubernetes objects in a Git repository.
+
+* Automated service deployment with https://crossplane.io/[Crossplane].
+* Backup of data with https://k8up.io[K8up] and https://restic.readthedocs.io[Restic].
+* GitOps with https://argoproj.github.io/argo-cd/[Argo CD].
+* Secrets management with https://www.vaultproject.io/[Vault].
+* Monitoring and alerting with Prometheus, Alertmanager and Signalilo.
+
+=== What's in for: Ops
+
+With Project Syn it gets easy to manage a fleet of Kubernetes clusters. Delivering applications and configuration to all managed Kubernetes clusters and providing an inventory of all the available clusters and their content is part of Project Syn.
+
+pass:[<!-- vale Microsoft.GenderBias = NO -->]
+
+* Configuration management for system components with xref:commodore::index.adoc[Commodore], https://kapitan.dev/[Kapitan] and https://jsonnet.org/[Jsonnet] by leveraging a hierarchical configuration value store.
+* Central cluster registry and inventory including GitOps Git repository management provided by https://github.com/projectsyn/lieutenant-api[Lieutenant API], https://github.com/projectsyn/lieutenant-operator[Lieutenant Operator] and xref:steward::index.adoc[Steward].
+* Automated component maintenance with https://github.com/renovatebot/renovate[Renovate] which helps to keep all the applications up-to-date.
+
+pass:[<!-- vale Microsoft.GenderBias = YES -->]
+
+=== High-Level Architecture
+
+The following diagram shows the current high-level architecture (subject to change):
+
+image::syn_highlevel_architecture.png[]
```

Comment:
Can we have that as an ASCII diagram to have it editable along with the documentation?

Reply:
We can, but I prefer to do this $later (to keep it lean ATM)